src/tests/ftest/container/attribute.py

```diff
@@ -46,20 +53,31 @@ def create_data_set():
             data_set[key] = get_random_bytes(size)
         return data_set
 
-    def verify_list_attr(self, indata, size, buff, mode="sync"):
+    def verify_list_attr(self, indata, outdata, mode="sync"):
         """
-        verify the length of the Attribute names
+        Args:
+            indata: Dict used to set attr
+            outdata: Dict obtained from list attr
+            mode: sync or async
+
+        Verify the length of the Attribute names
         """
-        length = sum(
-            len(i) if i is not None else 0 for i in list(indata.keys()))
-        length += len(list(indata.keys()))
+        length = 0
+        for key in indata.keys():
+            length += len(key)
+
         if mode == "async":
-            length = length + 1
+            length += 1
+
```

Comment:
Maybe some comments on attribute structure can be explained here.

Reply:
Sure, will add